src/index.ts

```diff
@@ -47,7 +47,15 @@ export default async function flow(options: Options = {}) {
     )
   )
 
-  const unflowedFiles = files.filter(({ content }) => !/@flow/.test(content)).map(({ path }) => path)
+  function detectFlowPragma(content): boolean {
+    const flowPragma = "@flow"
+    const maxLines = 10
```

Comment:
nit: this should read from `.flowconfig -> [options] -> max_header_tokens` https://flow.org/en/docs/config/options/#toc-max-header-tokens-integer and default to 10.

Reply:
Ohh that's a good idea, we should check whether the flowconfig has that option set for maximum compatibility!

Reply:
Wow, I had no idea. Thanks, Dustan!

Reply:
https://repl.it/@kevinSuttle/Parsing-flowconfig-to-find-maxheadertokens